concepticondata/conceptlists/Hsiu-2015-315.tsv

```diff
@@ -0,0 +1,317 @@
+Hsiu-2015-315-29	29	big	1202	BIG
+Hsiu-2015-315-30	30	bird	937	BIRD
+Hsiu-2015-315-31	31	bite	1403	BITE
+Hsiu-2015-315-32	32	bitter	887	BITTER
+Hsiu-2015-315-33	33	black	163	BLACK
+Hsiu-2015-315-34	34	Black Hmong [autonym]		
+Hsiu-2015-315-35	35	blind	1653	BLIND
+Hsiu-2015-315-36	36	blood	946	BLOOD
+Hsiu-2015-315-37	37	blunt	379	BLUNT
+Hsiu-2015-315-38	38	boat	1844	BOAT
+Hsiu-2015-315-39	39	bone	1394	BONE
+Hsiu-2015-315-40	40	bowl	1530	BOWL
+Hsiu-2015-315-41	41	brain	1583	BRAIN
+Hsiu-2015-315-42	42	breast	1402	BREAST
+Hsiu-2015-315-43	43	breathe	1407	BREATHE
+Hsiu-2015-315-44	44	buffalo, water	2516	WATER BUFFALO
+Hsiu-2015-315-45	45	burn	2102	BURN
+Hsiu-2015-315-46	46	bury	1719	BURY
+Hsiu-2015-315-47	47	butterfly	1791	BUTTERFLY
+Hsiu-2015-315-48	48	buy	1869	BUY
+Hsiu-2015-315-49	49	cat	1208	CAT
+Hsiu-2015-315-50	50	cattle; cow	1007	COW
+Hsiu-2015-315-51	51	cave	1155	CAVE
+Hsiu-2015-315-52	52	centipede	1974	CENTIPEDE
+Hsiu-2015-315-53	53	chair	1476	CHAIR
+Hsiu-2015-315-54	54	charcoal	1188	CHARCOAL
+Hsiu-2015-315-55	55	cheek	1730	CHEEK
+Hsiu-2015-315-56	56	chew	321	CHEW
+Hsiu-2015-315-57	57	chicken	1318	CHICKEN
+Hsiu-2015-315-58	58	child	2099	CHILD
+Hsiu-2015-315-59	59	chili pepper	919	CHILI PEPPER
+Hsiu-2015-315-60	60	chopstick	1987	CHOPSTICKS
+Hsiu-2015-315-61	61	citrus fruit	1126	CITRUS FRUIT
+Hsiu-2015-315-62	62	clean	704	CLEAN
+Hsiu-2015-315-63	63	close, to	1942	NEAR
+Hsiu-2015-315-64	64	cloud	1489	CLOUD
+Hsiu-2015-315-65	65	coin	1867	COIN
+Hsiu-2015-315-66	66	cold (adj.)	1287	COLD
+Hsiu-2015-315-67	67	copper	1825	COPPER
+Hsiu-2015-315-68	68	corn	506	MAIZE
+Hsiu-2015-315-69	69	cough, to	879	COUGH
+Hsiu-2015-315-70	70	cover, to	761	COVER
+Hsiu-2015-315-71	71	crossbow	3193	CROSSBOW
+Hsiu-2015-315-72	72	crow (bird)	913	CROW
+Hsiu-2015-315-73	73	cry, to	1839	CRY
+Hsiu-2015-315-74	74	cut, to	1432	CUT
+Hsiu-2015-315-75	75	dark	706	DARK
+Hsiu-2015-315-76	76	daughter	1357	DAUGHTER
+Hsiu-2015-315-77	77	day	1225	DAY (NOT NIGHT)
+Hsiu-2015-315-78	78	deaf	996	DEAF
+Hsiu-2015-315-79	79	deer	1936	DEER
+Hsiu-2015-315-80	80	dew	1977	DEW
+Hsiu-2015-315-81	81	die	1494	DIE
+Hsiu-2015-315-82	82	dig	1418	DIG
+Hsiu-2015-315-83	83	dirty	1230	DIRTY
+Hsiu-2015-315-84	84	dog	2009	DOG
+Hsiu-2015-315-85	85	door	1567	DOOR
+Hsiu-2015-315-86	86	dove	1853	DOVE
+Hsiu-2015-315-87	87	dream	2374	DREAM
+Hsiu-2015-315-88	88	drink water		
+Hsiu-2015-315-89	89	dry	1398	DRY
+Hsiu-2015-315-90	90	duck	1360	DUCK
+Hsiu-2015-315-91	91	dust	2	DUST
+Hsiu-2015-315-92	92	ear	1247	EAR
+Hsiu-2015-315-93	93	eat	1336	EAT
+Hsiu-2015-315-94	94	eat rice		
+Hsiu-2015-315-95	95	egg	744	EGG
+Hsiu-2015-315-96	96	elbow	981	ELBOW
+Hsiu-2015-315-97	97	embers	67	EMBERS
+Hsiu-2015-315-98	98	evening	1629	EVENING
+Hsiu-2015-315-99	99	eye	1248	EYE
+Hsiu-2015-315-100	100	face	1560	FACE
+Hsiu-2015-315-101	101	fall	1280	FALL
+Hsiu-2015-315-102	102	family	194	FAMILY
+Hsiu-2015-315-103	103	far	1406	FAR
+Hsiu-2015-315-104	104	fat	1282	FAT (FROM ANIMALS)
+Hsiu-2015-315-105	105	fear	1419	FEAR (BE AFRAID)
```

Comment:
Again, it may be ambiguous in this language, and specified in compounds, or it may be specific, so we need to check against the Hmong-Mien data by Chen.

Reply:
I know the pull request is closed, so my comments are all records for myself. 
I will keep this link. 怕 in Chen's concept list is linked to FEAR (BE AFRAID). The reflexes in Hsiu's data is "qhai⁵³". In Chen's data, there is "kʰei¹³" , "ʔhe³¹³" and "ha²²".

Reply:
So this is cognate then! At least it is highly likely, and the equivalent of "pà" in Chinese.